File: Code/SimpleBudget.API/Models/Taxes/TaxIncomeModel.cs
namespace SimpleBudget.API.Models
{
    public class TaxIncomeModel
    {
        public int PaymentId { get; set; }
        public string PaymentDate { get; set; } = default!;
        public string? CompanyName { get; set; }
        public string? Description { get; set; }
        public string? CategoryName { get; set; }
        public string WalletName { get; set; } = default!;
        public string CurrencyCode { get; set; } = default!;
        public decimal Rate { get; set; }
        public decimal Value { get; set; }
        public string ValueFormat { get; set; } = default!;
    }
}

File: Code/client/src/api/models/TaxIncomeModel.ts
export interface TaxIncomeModel {
    paymentId: number;
    paymentDate: string;
    companyName: string | undefined | null;
    description: string | undefined | null;
    categoryName: string | undefined | null;
    walletName: string;
    currencyCode: string;
    rate: number;
    value: number;
    valueFormat: string;
}